src/js/modules/displayTaskList.js

```diff
@@ -3,59 +3,84 @@ import checkBtnImg from '../../img/done.svg';
-    const deleteBtn = document.createElement('img');
-    deleteBtn.src = deleteImg;
-    deleteBtn.alt = 'delete icon';
-    deleteBtn.className = 'todo__btn-delete';
-
-    /* create a task drag button */
-    const dragBtn = document.createElement('img');
-    dragBtn.src = dragImg;
-    dragBtn.alt = 'delete icon';
-    dragBtn.className = 'todo__btn-drag';
-
-    /* Appending elements to parents */
-    taskCheckBtn.appendChild(taskCheckBtnImg);
-    taskDiv.appendChild(taskCheckBtn);
-    taskDiv.appendChild(todo);
-    taskDiv.appendChild(dragBtn);
-    taskDiv.appendChild(deleteBtn);
-    todoTastList.appendChild(taskDiv);
-  });
+  /* Checking if task list is empty */
+  if (list.length === 0) {
+    const emptyMsg = document.createElement('p');
+    emptyMsg.className = 'task__list-emply';
+    emptyMsg.innerText = 'Empty task list';
+    todoTastList.appendChild(emptyMsg);
+  } else {
+    /* loop through the task array */
+    list.forEach((task) => {
+      /* create a task div */
+      const taskDiv = document.createElement('div');
+      taskDiv.className = 'todo__item';
+
+      /* create a task check button */
+      const taskCheckBtn = document.createElement('button');
+      taskCheckBtn.className = 'todo__btn-check';
+
+      /* create a task check button image */
+      const taskCheckBtnImg = document.createElement('img');
+      taskCheckBtnImg.src = checkBtnImg;
+      taskCheckBtnImg.alt = 'to do completion icon';
+      taskCheckBtnImg.className = 'todo__btn-img';
+
+      /* create a task label */
+      const taskLabel = document.createElement('label');
+      taskLabel.className = 'todo__label';
+      taskLabel.htmlFor = `todo__item-${task.index}`;
+
+      /* create the todo element using textarea */
+      const todo = document.createElement('textarea');
+      todo.className = 'todo';
+      todo.name = 'todo_item';
+      todo.id = `todo__item-${task.index}`;
+      todo.cols = '30';
+      todo.rows = '1';
+      todo.innerText = task.description;
+
+      if (task.completed === true) {
+        todo.classList.add('completed');
+      }
+
+      /* create a task delete button */
+      const deleteBtn = document.createElement('button');
+      deleteBtn.className = 'todo__btn-delete';
+
+      /* create a task delete button image */
+      const deleteBtnImg = document.createElement('img');
+      deleteBtnImg.src = deleteImg;
+      deleteBtnImg.alt = 'delete icon';
+      deleteBtnImg.className = 'todo__delete-img';
+
+      /* create a task drag button */
+      const dragBtn = document.createElement('img');
+      dragBtn.src = dragImg;
+      dragBtn.alt = 'drag icon';
+      dragBtn.className = 'todo__btn-drag';
+
+      /* add event listener to the check button */
+      taskCheckBtn.addEventListener('click', () => {
+        taskCheckBtn.classList.toggle('checked');
+        taskCheckBtn.nextElementSibling.classList.toggle('completed');
+        task.completed = !(task.completed);
+      });
```

Comment:
@rloterh Thanks for pointing that out! I made a mistake with my class property naming. I used status instead of completed.

However, based on the project requirement, as shown below, 

> [By default new tasks should have the property `completed` set to `false`...](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#project-requirements) 

the piece of code you reference was not supposed to be there. I have updated the code and removed the piece of code.

![image](https://user-images.githubusercontent.com/22054464/202496627-6c23ff59-a88e-4101-97be-746454269f08.png)

Reply:
Hey @fmanimashaun, I think the previous reviewer checked for the interactivity between the checkboxes which is meant to be a requirement for the third milestone, however, your app work as expected on this milestone [Add and Remove] but there's an issue with the app as one can add empty todo as this is not the behavior you want. Please, kindly fix this so your add doesn't add on the empty field. I'd mark this as optional but kindly fix it because you it won't be marked as optional for the next project milestone.

Reply:
Hi @fmanimashaun @zieeco 
My remark was based on this highlighted-on part of the student [requirement for this project](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md), as you can see below:

![image](https://user-images.githubusercontent.com/12745474/202525017-63571481-4e48-43fc-a21e-c3243f911836.png)

I actually counted the _checking_ or _unchecking_ of the tasks to be part of the **All changes to the To Do List** which should accordingly be reflected in the local storage.

Perhaps, this aspect is meant to be implemented in the next module as you mentioned. In that case I apologize for missing out on that aspect. Cheers, and Happy coding! 😃